docs/docs/gatsby-on-linux.md

```diff
@@ -2,7 +2,7 @@
 title: Gatsby on Linux
 ---
 
-# Linux
+## Linux
```

Comment:
```suggestion
```
This heading is unnecessary, so it could be removed entirely. There is a PR to add information on this page as well. See #18263